woocommerce/woocommerce
# [ARCHIVED] Setup marketplace React skeleton layout

Relating to the WooCommerce In-App Marketplace. This PR sets up the base skeleton that can be populated with the individual components.

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



- Define SCSS variables that can be shared across all marketplace components
- Content component and associated styles
- Match the tabs styling to the design
- Footer component using SlotFills







### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout this branch
2. from the `plugins/woocommerce-admin` folder run the build tools (eg. `nvm use` and `pnpm start:hot`)
3. Visit WooCommerce > Marketplace > Extensions (eg. http://localhost:8888/wp-admin/admin.php?page=wc-admin&path=%2Fmarketplace&tab=extensions)
4. You should see the skeleton layout matches the designs and uses placeholder cards.
     - note: this doesn't currently include the search component and the header isn't here but there is space for it.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
